h5py/tests/test_h5d_direct_chunk.py

```diff
@@ -186,3 +186,75 @@ def test_fail_buffer_not_contiguous(self, writable_file):
         out = array[:, :, ::2]  # Array is not contiguous
         with pytest.raises(ValueError):
             dataset.id.read_direct_chunk((0, 0), out=out)
+
+
+@ut.skipIf(h5py.version.hdf5_version_tuple < (1, 14, 0),
+           "read_multi requires HDF5 >= 1.14.0")
+class TestReadMulti(TestCase):
```

Comment:
This is in the file `test_h5d_direct_chunk.py`, the rest of which is all about the direct chunk read/write methods. I think this file is small enough that we may as well just rename it to `test_h5d.py`.